tabs/motors.js

```diff
@@ -237,70 +257,160 @@ TABS.motors.initialize = function (callback) {
         });
 
         // set refresh speeds according to configuration saved in storage
-        chrome.storage.local.get('motors_tab_gyro_settings', function (result) {
-            if (result.motors_tab_gyro_settings) {
-                $('.tab-motors select[name="gyro_refresh_rate"]').val(result.motors_tab_gyro_settings.rate);
-                $('.tab-motors select[name="gyro_scale"]').val(result.motors_tab_gyro_settings.scale);
+        chrome.storage.local.get('motors_tab_sensor_settings', function (result) {
+            if (result.motors_tab_sensor_settings) {
+              var sensor = result.motors_tab_sensor_settings.sensor;
+              $('.tab-motors select[name="sensor_choice"]').val(result.motors_tab_sensor_settings.sensor);
```

Comment:
This needs a default setting for the case when `motors_tab_sensor_settings` is not defined, otherwise users will get no trace at all the first time they open the Motors tab.

Reply:
Thanks - left it hanging when moving things.